### PR TITLE
API-7030: translation properties

### PR DESCRIPTION
**Pull request by lpmichalik**
## 🚀 Links

- [Feature branch](https://developers.labs.livechat.com/docs/feature/API-7030-translation-props)
- [Jira](https://livechatinc.atlassian.net/browse/API-7030)

## ✅ Checklist (for API docs)

- Changelog
- API versions
- Web & RTM
- **Table:** Available methods/webhooks/pushes
- **Table:** Specifics

For more guidelines, see [Updating API docs](https://livechatinc.atlassian.net/wiki/spaces/PAT/pages/761529400/Updating+API+docs).

### Content

- New content
  
### Features (for the website)

- New feature

### CODE REVIEW

File: content/messaging/references/default-properties/index.mdx [hunk 1]
```diff
@@ -480,6 +481,60 @@ Use these properties to test things out and learn how properties work. [More on
...
+<Section>
+<Text>
+
+## translation
+
+These properties are used for storing translated text in message events.
+
+### source_lang_code
+
+Specifies source language code.
```
Comment (ale-kacperczyk) [inline content/messaging/references/default-properties/index.mdx:494]:
```suggestion
Specifies the source language code.
```

File: content/messaging/references/default-properties/index.mdx [hunk 2]
```diff
@@ -480,6 +481,60 @@ Use these properties to test things out and learn how properties work. [More on
...
+
+|               |                                                                                     |
+| ------------- | ----------------------------------------------------------------------------------- |
+| **Access**    | <ul><li>for Agents, `rw` in events</li> <li>for Customers, `r` in events </li></ul> |
+| **Data type** | `string`                                                                            |
+
+### target_lang_code
+
+Specifies target language code.
```
Comment (ale-kacperczyk) [inline content/messaging/references/default-properties/index.mdx:503]:
```suggestion
Specifies the target language code.
```

File: content/messaging/references/default-properties/index.mdx [hunk 3]
```diff
@@ -480,6 +481,60 @@ Use these properties to test things out and learn how properties work. [More on
...
+
+|               |                                                                                     |
+| ------------- | ----------------------------------------------------------------------------------- |
+| **Access**    | <ul><li>for Agents, `rw` in events</li> <li>for Customers, `r` in events </li></ul> |
+| **Data type** | `string`                                                                            |
+
+### target_message
+
+Message text in target language.
```
Comment (ale-kacperczyk) [inline content/messaging/references/default-properties/index.mdx:512]:
```suggestion
Contains message text in the target language.
```

**Review by ale-kacperczyk — CHANGES_REQUESTED** [review]

**Comment by bkubiak** [comment]
I can't find the update, what am I doing wrong?
https://developers.labs.livechat.com/docs/messaging/references/default-properties/#available-properties

**Comment by lpmichalik** [comment]
https://developers.labs.livechat.com/docs/feature/API-7030-translation-props/messaging/references/default-properties/#header

File: content/messaging/references/default-properties/index.mdx [hunk 4]
```diff
@@ -480,6 +481,60 @@ Use these properties to test things out and learn how properties work. [More on
...
+<Code>
+<CodeResponse title={'Properties in the translation namespace'}>
+
+```json
+{
+  "properties": {
+      "translation": {
+		  	"source_lang_code": "en",
+		  	"target_lang_code": "pl",
+			  "target_message": "Witaj Świecie!"
```
Comment (ale-kacperczyk) [inline content/messaging/references/default-properties/index.mdx:529]: indentation too big

**Review by bkubiak — APPROVED** [review]

**Review by ale-kacperczyk — APPROVED** [review]

**Pull request merged**